### DISCUSSION TITLE
Is there anybody knows the restapi access_token and refresh_token expiration time

### DISCUSSION THREAD

**Discussion by cw1427** [apache/superset#26844] [Q&A / Help]
Is there anybody knows the restapi access_token and refresh_token expiration time

**Comment by dpgaspar** [comment]
Superset uses: flask-jwt-extended, take a look at the defaults:

https://flask-jwt-extended.readthedocs.io/en/stable/options.html#JWT_ACCESS_TOKEN_EXPIRES
https://flask-jwt-extended.readthedocs.io/en/stable/options.html#JWT_REFRESH_TOKEN_EXPIRES

These can be changed on Superset config
